pkgs/development/haskell-modules/configuration-ghc-9.0.x.nix

```diff
@@ -104,14 +104,69 @@ self: super: {
   # https://github.com/Soostone/retry/issues/71
   retry = dontCheck super.retry;
 
-  # hlint 3.3 needs a ghc-lib-parser newer than the one from stackage
-  hlint = super.hlint_3_3_4.overrideScope (self: super: {
-    ghc-lib-parser = overrideCabal self.ghc-lib-parser_9_0_1_20210324 {
-      doHaddock = false;
-    };
-    ghc-lib-parser-ex = self.ghc-lib-parser-ex_9_0_0_4;
-  });
+  hlint = super.hlint_3_3_4;
 
   # pick right version for compiler
   ghc-api-compat = doDistribute super.ghc-api-compat_9_0_1;
+  ghc-lib-parser = overrideCabal self.ghc-lib-parser_9_0_1_20210324 {
+    doHaddock = false;
```

Comment:
Is there an issue  / context for this?

Reply:
I have no clue. It was workaround for something back in march, but it doesn‘t seem to be necessary anymore.